### PR TITLE
fix(metadata): added aria-expanded to collapsible buttons

### PR DESCRIPTION
**Pull request by msorianoBox**
Hi guys! 

One of the current issues we have is that some collapsible buttons doesn't give enough information to the users while they are being expanded or not.
Screen Shot 2021-09-21 at 9 44 26

Added the aria-expanded attribute so screen readers can inform the users while the buttons are open or not
Screen Shot 2021-09-21 at 9 45 01

### CODE REVIEW

File: src/components/collapsible/Collapsible.tsx [hunk 1]
```diff
@@ -102,6 +102,7 @@ class Collapsible extends React.PureComponent<CollapsibleProps, CollapsibleState
         const resinTagTarget: string = RESIN_TAG_TARGET;
         const modifiedButtonProps: { [index: string]: string } = omit(buttonProps, [resinTagTarget]);
         const interactionTarget = buttonProps[resinTagTarget];
+        const ariaExpanded = isOpen ? 'true' : 'false';
```
Comment (jstoffan) [inline src/components/collapsible/Collapsible.tsx:105]: Can we in-line this code with its prop since it isn't reused?

Comment (msorianoBox) [reply src/components/collapsible/Collapsible.tsx:105]: Will do! :D

Comment (msorianoBox) [reply src/components/collapsible/Collapsible.tsx]: Done!

File: src/components/collapsible/Collapsible.tsx [hunk 2]
```diff
@@ -117,6 +117,7 @@ class Collapsible extends React.PureComponent<CollapsibleProps, CollapsibleState
                         {...modifiedButtonProps}
                         className="collapsible-card-title"
                         onClick={this.toggleVisibility}
+                        aria-expanded={isOpen ? 'true' : 'false'}
```
Comment (jstoffan) [inline src/components/collapsible/Collapsible.tsx:120]: It looks like `aria-expanded` expects a boolean. Can use `aria-expanded={isOpen}` here? Also, can we keep the props alphabetized?

**Review by jstoffan — APPROVED** [review]

**Pull request merged**